llvm/docs/DeveloperPolicy.rst

```diff
@@ -539,11 +539,21 @@ What are the expectations around a revert?
 Obtaining Commit Access
 -----------------------
 
-We grant commit access to contributors that can provide a valid justification.
-If you would like commit access, please use this `link <https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20%3Cuser%3E&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
+Once you have 3 or more merged pull requests, you may use this
+link <https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20%3Cuser%3E&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
 an issue and request commit access. Replace the <user> string in the title
 with your github username, and explain why you are requesting commit access in
-the issue description. If approved, a GitHub invitation will be sent to your
+the issue description.  Once the issue is created, you will need to get two
+current contributors to support your request before commit access will be granted.
+
+Reviewers of your committed patches will automatically be CCed upon creating the issue.
+Most commonly these reviewers will provide the necessary approval, but approvals
+from other LLVM committers are also acceptable. Those reviewing the application are
+confirming that you have indeed had three patches committed, and that based on interactions
+on those reviews and elsewhere in the LLVM community they have no concern about you
+adhering to our Developer Policy and Code of Conduct.
+
+If approved, a GitHub invitation will be sent to your
 GitHub account. In case you don't get notification from GitHub, go to
 `Invitation Link <https://github.com/orgs/llvm/invitation>`_ directly. Once
 you accept the invitation, you'll get commit access.
```

Comment:
I can't comment on the lines below directly, but the following paragraph should probably be updated too:

> Prior to obtaining commit access, it is common practice to request that
someone with commit access commits on your behalf. When doing so, please
provide the name and email address you would like to use in the Author
property of the commit.

In fact, I think it can be removed entirely, but we may want to keep the bit of information about a valid author name and email address if we don't discuss it anywhere else.